Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM ros:noetic-ros-core
+
+SHELL ["/bin/bash", "-c"]
+
+WORKDIR /ros_ws
+
+RUN apt-get update  && \
+    apt-get install -y \
+        git \
+        python3-pip && \
+    pip3 install \
+        rosdep && \
+    git clone -b decode-udp-line https://github.com/ros-drivers/nmea_navsat_driver.git src/nmea_navsat_driver && \
```

Comment:
Is there a particular reason we use this branch?

Reply:
Regarding the [documentation](https://husarion.com/manuals/panther/#software-guide), it is recommended to use this branch. This is the only properly working version of this software.

Reply:
ok